### ISSUE TITLE
[Buttons] Voiceover does not announce grayed out button as dimmed

### ISSUE THREAD

**Issue by material-mirror** [material-components/material-components-ios#6944]
This was filed as an internal issue. If you are a Googler, please visit [b/128893463](http://b/128893463) for more details.

---
#### Internal data
- Associated internal bug: [b/128893463](http://b/128893463)

**Comment by romoore** [comment]
Should remain "open" until the release lands in https://github.com/material-components/material-components-ios/issues/6944. This is a results of the mirroring bot attempting to close the internal issue before internal clients can make use of the fix.
